Meta/run.sh

```diff
@@ -236,13 +236,15 @@ if [ -z "$SERENITY_MACHINE" ]; then
     fi
 fi
 
-if [ "$NATIVE_WINDOWS_QEMU" -ne "1" ]; then
+# Enable QEMU QMP protocol only if we want to profile the kernel
+# via QMP. Enabling this otherwise removes the QEMU monitor, which
+# is required to debug things like USB
+if [ "$NATIVE_WINDOWS_QEMU" -ne "1" ] && [ "$SERENITY_PROFILE_KERNEL_QMP" -eq "1" ]; then
```

Comment:
I haven't touched this script in a long, _long_ time (possibly 2019), so someone let me know if `$SERENITY_PROFILE_KERNEL_QMP` needs to be plonked somewhere else besides here.